integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
You are right. Let me rephrase your words a bit. There are 2 problems:

1. IT doesn't use `livy-scala-api`.
2. test-lib is not built with Scala 2.10 and 2.11.

I think the correct fix here is changing `livy-test-lib` to build with Scala 2.10 and 2.11.